src/test/java/gov/nasa/pds/validate/ValidationIntegrationTests.java

```diff
@@ -480,23 +480,6 @@ void testGithub09() {
 
             assertEquals(count, 0, ProblemType.FIELD_VALUE_DATA_TYPE_MISMATCH.getKey() + " info/error messages not expected.");
             
-            report = new File(outFilePath + File.separator + "report_github09_3.json");
-            String[] args4 = {
-                    "-r", report.getAbsolutePath(),
-                    "-s", "json",
-                    "-t" , testPath + File.separator + "csv_empty_field_test_INVALID.xml",
-                    
-                    };
-            this.launcher = new ValidateLauncher();
-            this.launcher.processMain(args4);
-
-            gson = new Gson();
-            reportJson = gson.fromJson(new FileReader(report), JsonObject.class);
-
-            count = this.getMessageCount(reportJson, ProblemType.FIELD_VALUE_DATA_TYPE_MISMATCH.getKey());
-
-            assertEquals(count, 1, ProblemType.FIELD_VALUE_DATA_TYPE_MISMATCH.getKey() + " info/error messages expected.");
-
```

Comment:
@hhlee445 this integration test is actually valid and indicates the solution will unfortunately not work. Note in the first row of data for [the file](https://raw.githubusercontent.com/NASA-PDS-Incubator/validate/master/src/test/resources/github09/csv_empty_field_test_INVALID.csv), one of the columns is supposed to be an ASCII_Integer, but is just a space. That is actually invalid and was actually what #9 was all about. You can have spaces surround and integer, but it can't be only whitespace.

```
13, Plains,First_Soil,Soil,First_Soil1_Final,,0,0,13,, ,3,9,"0.406622, -0.859749, 0.275765","0.095748, -0.031037, -0.994659",,,FirstSoil1Final ,
14, Plains,First_Soil,Soil,First_Soil1_Final,,0,0,,14,14,3,9,"0.406622, -0.859749, 0.275765","0.095748, -0.031037, -0.994659",Gusev_Soil,FirstSoil1Final,,

...13,, ,3...
-------^-----
...14,14,3,9...
---------^-----
```